src/components/Dropdown/Items/MultiSelectItem/MultiSelectItem.tsx

```diff
@@ -25,6 +24,7 @@ export interface Props<PriorityItemValue> extends DropdownMenuCheckboxItemProps
 const { block } = bem('MultiSelectItem', styles);
```

Comment:
It would be nice to have story with new pill and content dropdown to see how it works together.

Reply:
there is a story. you just have to choose the variant in the controls. Or is there something else?

Reply:
yes, I also mean that :D 
<img width="745" alt="Screenshot 2024-11-08 at 12 07 59" src="https://github.com/user-attachments/assets/7549a965-0c53-4458-8f13-3d51bc7815b6">

Reply:
hmm, it's strange. Maybe something wrong on my machine. I have the error:
![Screenshot 2024-11-08 at 12 20 29](https://github.com/user-attachments/assets/62c2287c-3e30-4f5b-a34a-0e4e41e4feae)
@AnastasiiaMP could you please check if it works for you?

Reply:
Pill has dropdown only in one story - with the name `Pill`, in all other stories this error is happening to me as well

Reply:
added the missing argument to the stories

Reply:
Can you try now, I think you were looking before I pushed the changes.

Reply:
I don't have the errors now, but I still don't see the dropdown. Could you please check @AnastasiiaMP ?

Reply:
@mukiienko are you looking at this story? https://oneui-e9xydbbbf-textkernel-oneuis-projects.vercel.app/?path=/story/molecules-pill--pill&args=variant:enhanced